core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -80,7 +80,9 @@
   protected boolean mClosed;
   /** Whether or not the current block should be cached. */
   protected boolean mShouldCacheCurrentBlock;
-  /** Current position of the stream. */
+  /** Include incomplete blocks if Alluxio is configured to store blocks in Alluxio storage. */
+  private boolean mShouldCacheIncompleteBlock;
```

Comment:
maybe Incomplete -> PartialRead

also, how does this variable interact with `mShouldCacheCurrentBlock`?